common/changes/@microsoft/api-extractor/condense-tokens_2022-07-31-14-58.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "Condense ExcerptTokens in post-processing step after original construction instead of attempting to do this mid-construction",
```

```suggestion
      "comment": "Fix a regression where .api.json excerpts were sometimes missing tokens (GitHub #3561) and generally improve the quality of excerpt generation",
```

Change log messages should describe what problem we solved for end users. They probably aren't interested in how we implemented it. 😉